web/app/js/components/TopRoutesTable.jsx

```diff
@@ -10,6 +10,11 @@ const routesColumns = [
     dataIndex: "route",
     sorter: (a, b) => (a.route).localeCompare(b.route)
   },
+  {
+    title: "Service",
```

Comment:
Let's call this column "Authority", and if it's straightforward, can we also add a tooltip explaining what an authority is?